runtime/reconcilers/explore.go

```diff
@@ -127,10 +127,54 @@ func (r *ExploreReconciler) validateAndRewrite(ctx context.Context, self *runtim
 		return nil, fmt.Errorf("parent metrics view %q is invalid", spec.MetricsView)
 	}
 
-	// Add the access and field access security rules from the parent metrics view.
-	for _, rule := range mv.SecurityRules {
-		if rule.GetAccess() != nil || rule.GetFieldAccess() != nil {
-			spec.SecurityRules = append(spec.SecurityRules, rule)
+	if len(spec.SecurityRules) == 0 {
+		for _, rule := range mv.SecurityRules {
+			if rule.GetAccess() != nil || rule.GetFieldAccess() != nil {
+				spec.SecurityRules = append(spec.SecurityRules, rule)
+			}
+		}
+	} else {
+		for _, rule := range mv.SecurityRules {
+			if rule.GetFieldAccess() != nil {
+				spec.SecurityRules = append(spec.SecurityRules, rule)
+			}
```

Comment:
We either need to:
1. Handle merge of `FieldAccess` rules
2. Or, return an error if someone uses a `FieldAccess` rule on an explore (can return an error from the reconciler here if we find one)
    - So basically we only allow a basic `Access` rule on explores